python/tchannel/statsd.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2015 Uber Technologies, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from __future__ import absolute_import
+
+from .event import EventHook
+from .messages.error import ErrorMessage
+from .tornado.response import StatusCode
+
+
+class StatsdHook(EventHook):
+    """Collect Statsd information in the tchannel req/resp."""
+
+    def __init__(self, statsd):
+        """
+
+        :param statsd: instance of StatsD.
+            `Link text https://github.com/etsy/statsd`
```

Comment:
What? You probably meant:

    instance of `statsd <https://github.com/etsy/statsd>`_ 

The `<`, `>`, and `_` are not optional. `Link text` refers to the text you want to appear in place of the link.